__tests__/app.test.js

```diff
@@ -7,6 +7,11 @@ const seed = require("../db/seeds/seed");
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
```

Comment:
Cool idea here to make sure we don't have any console.logs left in the code. We could use the beforeEach and afterAll we already have to keep this a bit cleaner and more DRY>

Reply:
Good call - I got in a mess here for a while with the tests hanging, then realised that I needed to return the `seed()`